src/navigation/BackButton.js

```diff
@@ -4,12 +4,12 @@ import type { Node } from "react";
 import React from "react";
 import colors from "styles/tailwindColors";
 
-const BackButton = (): Node => {
+const BackButton = ( { tintColor = colors.black } ): Node => {
```

Comment:
A little out of scope, but I think this custom BackButton should live in `SharedComponents/Buttons` so we know to reuse it throughout the app. Right now we're using HeaderBackButton in ~5 places